src/System.Net.Sockets/tests/FunctionalTests/DualModeSocketTest.cs

```diff
@@ -2733,8 +2734,9 @@ private void ClientSend(object state, bool redundant)
                         socket.SendToAsync(e);
                     }
                 }
-                catch (SocketException)
+                catch (SocketException e)
                 {
+                    Console.Error.WriteLine(e.ToString());
```

Comment:
If you can please write to your ITestOutputHelper passed in above instead of using console writes. Writing to console is spammy and can't be associated easily with the test that wrote it.
here's info:
http://xunit.github.io/docs/capturing-output.html

Reply:
The output appears in the xml log, which is accessible through mission control UI